app/Http/Controllers/admin/QuestionController.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace App\Http\Controllers\admin;
+
+use Illuminate\Http\Request;
+use App\exam;
+use Session;
+use App\question;
+use App\Http\Requests;
+use App\Http\Controllers\Controller;
+
+class QuestionController extends Controller
+{
+     public function __construct()
+    {
+        $this->middleware('auth');
+    }
+
+    public function index()
+    {
+        $result = question::paginate(5);
+        return view('')->with('result',$result);
+    }
+
+    public function create()
+    {
+    	 $result = exam::get();
+        return view('admin.add_question')->with('result',$result); 
+    }
+
+    public function createQuestion($id)
```

Comment:
What is different between create and createQuestion. remove one of method if it is not required.